linalg/hypre.cpp

```diff
@@ -1018,6 +1034,47 @@ HypreParMatrix * HypreParMatrix::Transpose() const
    return new HypreParMatrix(At);
 }
 
+#if MFEM_HYPRE_VERSION >= 21800
+HypreParMatrix * HypreParMatrix::ExtractSubmatrix(Array<int> &indices,
```

Comment:
This function does not appear to be used anywhere in this PR. Is it generally useful enough to be included?

Reply:
It is useful for preconditioning on subdomains and worth having access to